pkg/installation/utils_test.go

```diff
@@ -216,6 +216,53 @@ func Test_ReplaceDockerImageURL(t *testing.T) {
 	}
 }
 
+func Test_InsertProfile(t *testing.T) {
+	t.Parallel()
+	const replacedWithData = "evaluation"
+	testData := []struct {
+		testName       string
+		data           File
+		expectedResult File
+		shouldFail     bool
+	}{
+		{
+			testName: "correct data test",
```

Comment:
Maybe we should also have a test case for incorrect data.

Reply:
Done.